Zend/tests/accessors/unknown_accessor.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+Unknown accessor
+--FILE--
+<?php
+
+class Test {
+    public $prop {
+        foobar {}
+    }
+}
+
+?>
+--EXPECTF--
+Fatal error: Unknown accessor "foobar" for property Test::$prop in %s on line 5
```

Comment:
Would it make more sense to include details on expected tokens to make it easier to fix typos or understand the error?

```suggestion
Fatal error: Unknown accessor "foobar" for property Test::$prop (expected "get" or "set") in %s on line 5
```